tests/agg_test.py

```diff
@@ -509,3 +509,11 @@ def test_agg_count_with_custom_name():
     df_grouped = df.groupby(df.x, sort=True).agg({'mycounts': vaex.agg.count(), 'mycounts2': 'count'})
     assert df_grouped['mycounts'].tolist() == [3, 2, 1]
     assert df_grouped['mycounts2'].tolist() == [3, 2, 1]
+
+
+def test_minmax_ts(df):
```

Comment:
I'd prefer that in addition to this, the test asserts for the min and max against the actual min and max values. It could happen that `minmax` is consistent with `min`  and `min`  but still objectively wrong.

Reply:
Good point. Not sure why I didn't have that check in the first commit.